### PR TITLE
fix: use hourCycle for Intl.DateTimeFormats

### PR DESCRIPTION
**Pull request by 0xdhrv**
## Summary

- This fixes the regex replace for 24:00 issue using hour cycle 

### References
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#hourcycle
- https://github.com/muan/site/blob/a07a84bffd8105fe350791991f24d1f9149f9d4f/assets/site.js#L5-L15C29

### CODE REVIEW

**Comment by 0xdhrv** [comment]
@muan if can review this

File: assets/site.js [hunk 1]
```diff
@@ -5,15 +5,13 @@ function settime() {
   const options = {
     timeZone: "Europe/Berlin",
     timeStyle: "short",
-    hour12: false
+    hour12: false,
+    hourCycle: "h23"
```
Comment (muan) [inline assets/site.js:9]:
```suggestion
    hour12: false
```

According to MDN,

> When true, this option sets hourCycle to either "h11" or "h12", depending on the locale. _When false, it sets hourCycle to "h23"._

**Review by muan — CHANGES_REQUESTED** [review]
Sorry for the huge delay! I remember seeing this but it slipped through the cracks. Just one requested change. Thank you!

**Review by muan — APPROVED** [review]

**Comment by muan** [comment]
Thank you!

**Pull request merged**